doc/srcspan.md

```diff
@@ -0,0 +1,124 @@
+# Goal
+
+We want our debugger to provide user information at runtime about which
+location the currently-executing UPLC code was "sourced" from.
+Such a location is called a `SrcSpan` and is `(haskellHsFilename,startline,startcol,endline,endcol)`.
+
+There are two types of `SrcSpan` locations:
+
+## i. definition-site locations
+
+All GHC_Core (type) variables come by default annotated with a `SrcSpan` that points
```

Comment:
Can you include where to find this - is it a field of `data Var`, or in a `Tickish`, or somewhere else?